### ISSUE TITLE
Linux: "standard" way (PPA, .deb) to install NanoVNA-Saver?

### ISSUE THREAD

**Issue by Ho-Ro** [NanoVNA-Saver/nanovna-saver#517]
**See [this request](https://groups.io/g/nanovna-users/topic/92215641#28696) from NanoVNA user group:**
*"Hello, is there possible to use a "standard" way (PPA, .deb) to install NanoVNA-Saver?
Thanks"*

**Describe the solution you'd like**
I build a deb package with this simple command:
```
DEB_BUILD_OPTIONS=nocheck python3 setup.py --command-packages=stdeb.command bdist_deb
```
This command (with some improvements) could go into a `Makefile` to ease the build and install process for Linux users.
```Makefile
# build a new debian package and link it in the current directory
.PHONY: deb
deb:
        DEB_BUILD_OPTIONS=nocheck python3 setup.py \
        --command-packages=stdeb.command \
        sdist_dsc --compat 10 --section electronics bdist_deb
        -rm python3-nanovnasaver_*.deb
        -ln `ls deb_dist/python3-nanovnasaver_*.deb | tail -1` .


# build and install a new debian package
.PHONY: debinstall
debinstall: deb
        sudo apt install ./python3-nanovnasaver_*.deb


# remove all editor artefacts
.PHONY: clean
clean:
        -rm *~ .*~


# remove all package build artefacts
.PHONY: distclean
distclean:
        python setup.py clean
        -rm -rf *~ .*~ deb_dist dist *.tar.gz *.egg* *.deb


```
You should also add some entries to`.gitignore`
```
/deb_dist/
*.deb
*.tar.gz
```
You need the debian package `python3-stdeb` (Python to Debian source package conversion plugins for distutils) and maybe some more development packages. 
For a similar project with github actions I do this build setup:
```
sudo apt-get install --quiet --yes fakeroot python3-setuptools python3-stdeb dh-python
```
As a quick start see [this documentation](https://pypi.org/project/stdeb/).

**Comment by Ho-Ro** [comment]
Ok, @zarath would you like me to provide a PR? It's a simple enhancement for Debian (and also Ubuntu) users without destroying the Win etc. functionality.

**Comment by zarath** [comment]
with pleasure

**Comment by Ho-Ro** [comment]
Ok, implemented with #560 and #662 - ca be closed.
